src/MSALWrapper/PCACache.cs

```diff
@@ -37,20 +38,26 @@ internal class PCACache
         /// </summary>
         /// <param name="logger">The logger.</param>
         /// <param name="tenantId">The tenant id.</param>
-        /// <param name="cacheFilePath">The cache file name.</param>
-        /// <param name="osxKeyChainSuffix">The osx key chain suffix.</param>
-        internal PCACache(ILogger logger, Guid tenantId, string cacheFilePath, string osxKeyChainSuffix = null)
+        internal PCACache(ILogger logger, Guid tenantId)
         {
             this.logger = logger;
-            this.osxKeyChainSuffix = string.IsNullOrWhiteSpace(osxKeyChainSuffix) ? $"{tenantId}" : $"{osxKeyChainSuffix}.{tenantId}";
+            this.osxKeyChainSuffix = $"{OSXKeyChainCategory}.{tenantId}";
 
-            if (string.IsNullOrWhiteSpace(cacheFilePath))
+            this.cacheFileName = $"msal_{tenantId}.cache";
+            this.cacheDir = this.CacheServiceFolder;
+        }
+
+        /// <summary>
+        /// Gets the absolute path of the cache folder. Only available on Windows.
+        /// </summary>
+        private string CacheServiceFolder
+        {
+            get
             {
-                throw new ArgumentNullException($"{nameof(cacheFilePath)} should not be null or whitespace.");
+                string appData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+                string absolutePath = Path.Combine(appData, ".IdentityService");
+                return absolutePath;
             }
-
-            this.cacheFileName = Path.GetFileName(cacheFilePath);
-            this.cacheDir = Directory.GetParent(cacheFilePath).FullName;
         }

```

Comment:
Last recommendation, Typically Properties don't require computation. I don't think there is a good reason to have this be a property instead of a function. We are effectively memoizing the value into a member of the class on construction - so I'd prefer if we make this a method, instead of a Property.